### ISSUE TITLE
Could not install Rb.Forms.Barcode 0.7.1 

### ISSUE THREAD

**Issue by ericbrunner** [rebuy-de/rb-forms-barcode#60]
Hi,

I just wanted to install the nugets in PCL , Droid and iOS project and get that install error:

> Could not install package 'Rb.Forms.Barcode 0.7.1'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile259', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.			

Hope you can help my out.
thanks in advance

Eric

**Comment by fvermeulen** [comment]
Do you try it to install from a windows system?

If that is the case we all not working with windows systems and only able to try it out on mac. So you could help us to get it working on a windows system

Here is the other issue from windows users
https://github.com/rebuy-de/rb-forms-barcode/issues/57

**Comment by ericbrunner** [comment]
@fvermeulen Yes Visual Studio 2015 Enterprise including latest stable Nuget Package Manager, Xamarin SDK and SP. I guess the PCL profile has to be changes on my side. But to which one?

**Comment by ericbrunner** [comment]
I digged through the source code and saw that your PCL i configured for that profile

> TargetFrameworkProfile>Profile78</TargetFrameworkProfile>

I will try to change my project, if no side effects popup!

**Comment by fvermeulen** [comment]
@ericbrunner I use PCL-Profile7 and with that it works as well fine, but which one do you used?

**Comment by ericbrunner** [comment]
@ I develop an Azure Mobile Apps Xamarin.Forms PCL App for Droid + iOS . The initial version was scaffolded by the Azure Portal with Profile=259.

**Comment by ericbrunner** [comment]
@fvermeulen Well I tried to switc hto Profile7 but I get several compiler errors which is clear becasue the nugets are not compatible anymore. Ok I understand that you nuget is not compatible to Profile259 now.
Sorry, I will use ZXing.Net which is working.

**Comment by fvermeulen** [comment]
yeah you have to retarget your packages. But that is possible. In Xamarin Studio you have to press a button for that. I don't know how that will be handled in Visual Studio.

I looked what is PCL259 for and PCL 7. In your case it makes more sense to use PCL 7, but anyway sorry that I cannot help you with your problem

**Comment by ericbrunner** [comment]
@fvermeulen Could you send my a link or screenshot where I can do that migration in Xamarin Studio? Then I would copy my code temp. from TFS to MAC , switch to Profile7 and copy it back to my windows box.

**Comment by rdoubleui** [comment]
@ericbrunner after changing your profile right-click on the `Packages` folder and select `Retarget`.

![screen shot 2016-12-21 at 08 28 20](https://cloud.githubusercontent.com/assets/542736/21380788/db6664fc-c757-11e6-85a0-1bab26439fc9.png)

**Comment by ericbrunner** [comment]
@rdoubleui Hi Thanks. I tried but I have no Retarget Command. I user the latest Xamarin Studio Community Edition. Do I have to install some Tools ?

![xs_ios](https://cloud.githubusercontent.com/assets/6480047/21381408/6a7f7aa4-c75b-11e6-8479-cea56ccb5532.jpg)

**Comment by rdoubleui** [comment]
The feature has been there from [version 5.2](https://developer.xamarin.com/releases/studio/xamarin.studio_5.2/xamarin.studio_5.2/), don't think that there's a different behaviour within the community edition. What nuget version are you using? I recall you need to have at least nuget 2.1

**Comment by rdoubleui** [comment]
@ericbrunner were you able to solve the issue with XS?

**Comment by ericbrunner** [comment]
@rdoubleui I currently try to retarget to `Profile=7`. I get back to you when I succeed.

**Comment by collaorodrigo7** [comment]
Do we have any progress with this? I am having the same problem

**Comment by collaorodrigo7** [comment]
So I changed it to profile 78 for adding the package, and then change it back to 111. I tested the demo the same UI from the sample on the repo and seems to be working just fine..
I dont know if changing it back to profile 111 is a good idea or no, hope someone can give an opinion on that.
Nice package, by the way, works pretty good!.

**Comment by navkar** [comment]
![image](https://user-images.githubusercontent.com/7309326/32579663-d32c32ac-c496-11e7-84f6-18f8b7d35bde.png)


See the imports section below. This perfectly works for me using .net standard. 

{
  "dependencies": {
    "Acr.UserDialogs": "6.5.1",
    "Ansuria.XFGloss": "1.0.6.59",
    "CarouselView.FormsPlugin": "4.4.6",
    "FreshEssentials": "2.1.3",
    "Humanizer": "2.2.0",
    "libphonenumber-csharp": "8.8.4",
    "Microcharts.Forms": "0.8.3-pre",
    "Microsoft.AspNet.SignalR.Client": "2.2.2",
    "Microsoft.Azure.Mobile.Analytics": "0.16.0",
    "Microsoft.Azure.Mobile.Crashes": "0.16.0",
    "Microsoft.Azure.Mobile.Push": "0.16.0",
    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
    "modernhttpclient.standard": "2.4.2",
    "NETStandard.Library": "2.0.1",
    "Newtonsoft.Json": "10.0.3",
    "PCLStorage": "1.0.2",
    "Plugin.Permissions": "2.1.0",
    "Plugin.Share": "7.0.1",
    "Plugin.VersionTracking": "2.0.1",
    "Rb.Forms.Barcode": "0.7.1",
    "Refractored.MvvmHelpers": "1.3.0",
    "Rg.Plugins.Popup": "1.0.4",
    "SkiaSharp": "1.59.1",
    "SkiaSharp.Views.Forms": "1.59.1.1",
    "Splat": "2.0.0",
    "SQLite.Net.Async-PCL": "3.1.1",
    "SQLite.Net.Core-PCL": "3.1.1",
    "Toasts.Forms.Plugin-PCL": "3.2.0",
    "Xam.Plugin.Connectivity": "3.0.3",
    "Xam.Plugin.DeviceInfo": "3.0.2",
    "Xam.Plugin.ExternalMaps": "4.0.1",
    "Xam.Plugin.Geolocator": "4.1.2",
    "Xam.Plugin.Hud": "1.0.5",
    "Xam.Plugin.Media": "3.0.1",
    "Xam.Plugins.Forms.ImageCircle": "1.8.1",
    "Xam.Plugins.Messaging": "4.0.0",
    "Xam.Plugins.Notifier": "2.1.0",
    "Xam.Plugins.Settings": "3.1.1",
    "Xamarin.Controls.SignaturePad.Forms": "2.2.0",
    "Xamarin.FFImageLoading": "2.2.22",
    "Xamarin.FFImageLoading.Forms": "2.2.22",
    "Xamarin.FFImageLoading.Transformations": "2.2.22",
    "Xamarin.Forms": "2.4.0.282",
    "Xamarin.Forms.Maps": "2.4.0.282",
    "Xamarin.Forms.Pages": "2.4.0.282",
    "Xamarin.Forms.Theme.Base": "1.0.0.43-pre1",
    "Xamarin.Forms.Theme.Dark": "1.0.0.43-pre1",
    "Xamarin.Forms.Theme.Light": "1.0.0.43-pre1"
  },
  "frameworks": {
    "netstandard1.4": {
      "imports": [
        "portable-net45+win81+wpa81+wp8",
        "portable-net45+win8+wp8"
      ]
    }
  }
}

**Comment by kam0u** [comment]
In my case I changed my project to profile 78 and still the package cannot be added, still having the same problem. The NuGet package is getting installed on Android and iOS projects but not on PCL

Windows System
Visual Studio 2017
Xamarin Forms 2.5.0.91635
Microcharts.Forms 0.7.1
